.github/workflows/nightly-builds.yaml

```diff
@@ -11,16 +11,17 @@ on:
         required: false
         default: 'v1.33.x'
       nightly_bucket:
-        description: 'Nightly bucket for builds'
+        description: 'Oracle Cloud bucket name for builds'
         required: false
-        default: 'gs://tekton-releases-nightly/pipeline'
```

Comment:
Is there a particular reason for removing the repo name from the bucket config and making it a separate param?

This pipeline should always publish to `<bucket>/pipeline` so making it configurable seems unusual, unless we're planning to use this as a template for a generic pipeline that can be reused across multiple repos. In which case, I think there are likely many other changes that would be required so each repo can customise the pipeline to their needs, and would require careful design before implementing.